azurerm/data_source_compute_resource_sku_test.go

```diff
@@ -0,0 +1,36 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceAzureRMComputeResourceSku_basic(t *testing.T) {
+	dataSourceName := "data.azurerm_compute_resource_sku.test"
+	location := testLocation()
+	name := "Standard_DS2_v2"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMComputeResourceSku_basic(name, location),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(dataSourceName, "name"),
```

Comment:
We should check that the name matches what we expect
```suggestion
					resource.TestCheckResourceAttr(dataSourceName, "name", name),
```

Reply:
An check that some of the other properties are set